pre-docs/inventory_version2.md

```diff
@@ -260,11 +259,67 @@ targets:
       transport: ssh
       ssh:
         user: root
-        password: 
+        password:
           _plugin: prompt
           message: please enter your ssh password
 ```
 
+#### pkcs7 plugin
+
+This plugin allows config values to be stored in encrypted in the inventory
+file and decrypted only as needed.
+
+`_plugin`: The value of `_plugin` must be `pkcs7`
+`encrypted-value`: The encrypted value. Generate encrypted values with `bolt secret encrypt <plaintext>`
+
+Example
+
+```
+version: 2
+targets:
+  - uri: 192.168.100.179
+    config:
+      transport: ssh
+      ssh:
+        user: root
+        password:
+          _plugin: pkcs7
+          encrypted-value: |
+                ENC[PKCS7,MIIBeQYJKoZIhvcNAQcDoIIBajCCAWYCAQAxggEhMIIBHQIBADAFMAACAQEw
+                DQYJKoZIhvcNAQEBBQAEggEAdCVkiddtK8jHz4g1y1pkB27VHCZx7dVzEiyT
+                33BgFv9atk8Ns/WE1tveFvyuEaDpk9y/FKisuh8DsTnR2mfGvHtX+BQdNqV6
+                L8/nIdwoEqYFd5sKFJnOlpdm7BMX4QDoCfGb+b2UB8A/7eJJ5AcgBVtrJLLE
+                VvqSCtqME12ltifdMivMP1hnVJOAhIpib8CwOIIP+Dtv7P7cPaHGTdQpR6Dp
+                jbe+AUDM6kcKGADLOYriPQ1UV6zDz5aeUbrwbr4FicHL/sQBPDcWIJR2elwY
+                bh8hCDe/IIWE7TOiauXOPyMPKohz622KNoJDJbmv5MhBwNFHSjgKAlOAxL3i
+                DK7XXzA8BgkqhkiG9w0BBwEwHQYJYIZIAWUDBAEqBBCvjDMKTjsHloKP04WO
+                Dq0ogBAUjTZMjbKjkndMSqPC5mGC]
+```
+
+Before using the pkcs7 plugin you need to create encryption keys. You can
+create these keys automatically with `bolt secret createkeys` or reuse existing
+hiera-eyaml pkcs7 keys with bolt secret. You can then encrypt values with `bolt
+secret encrypt <plaintext>` command and copy the result into your inventory
+file. If you need to inspect an encrypted value from the inventory you can
+decrypt it with `bolt secret decrypt <encrypted-value>`.
+
+##### Configuration
+
+By default keys are store in the `keys` directory of bolt project repo. If
```

Comment:
```suggestion
By default keys are stored in the `keys` directory of bolt project repo. If
```

Reply:
Also, I know you did not introduce this problem, but the options `keysize`, `private-key` etc appear to be renered on the same line in markdown. Not sure it that matters for easyditta given nobody complained about it for the other plugins.